test/unit/liquidity_pool_endpoints_test.js

```diff
@@ -140,5 +140,46 @@ describe('/liquidity_pools tests', function() {
           .catch(done);
       });
     });
+
+    const poolResponse = {
```

Comment:
🤔  What do you think of moving these const declarations inside the `it('filters by specific ID', ...`, to make it clearer what they were created for?

Reply:
Sure! To me, the way I "read" the tests is "here's a response, and here's the test that expects it". However, I think it's just as valid to read it as "here's a test, and here's the response it'll test against" as you're suggesting.

I can adjust this as part of #689 just so that there's less merge conflict work after a the rebase.

Reply:
Nvm, done in 848d054.